packages/cerebral-website-tutorial/parts/01/public/blaze.min.css

```diff
@@ -0,0 +1 @@
color:#fff;border-color:#2c3e50}.c-calendar__date--selected.c-button--active,.c-calendar__date--selected:hover.c-button--active{background-color:#1c2732}.c-calendar__date--selected:hover:not(:disabled):hover,.c-calendar__date--selected:not(:disabled):hover{background-color:#3c556e}.c-calendar__date--selected:hover:not(:disabled):focus,.c-calendar__date--selected:not(:disabled):focus{border-color:#2196f3;box-shadow:inset 0 0 0 2px #4dabf5}.c-calendar__date--selected:hover:not(:disabled):active,.c-calendar__date--selected:not(:disabled):active{background-color:#1c2732}.c-nav{background-color:#111;color:#fff;width:100%;margin:0;padding:0;z-index:3}.c-nav__content,.c-nav__item{display:block;height:3.5em;padding:0 1em;color:inherit;line-height:3.5em;vertical-align:middle}.c-nav__content .o-image,.c-nav__item .o-image{height:100%}.c-nav__item{text-decoration:none;cursor:pointer}.c-nav__item:not(:disabled):hover{background-color:#7b929e;color:#fff}.c-nav__item:not(:disabled):focus{box-shadow:inset 0 0 0 2px #4dabf5}.c-nav__item:not(:disabled):active{background-color:#647c88;color:#fff}.c-nav--inline .c-nav__content,.c-nav--inline .c-nav__item{display:inline-block}.c-nav--inline .c-nav__content--right,.c-nav--inline .c-nav__item--right{float:right}.c-nav--light{background-color:#f2f2ea;color:#3f2d26}.c-nav--top{top:0;bottom:auto}.c-nav--bottom,.c-nav--top{position:absolute;right:0;left:0}.c-nav--bottom{top:auto;bottom:0}.c-nav--left{right:auto;left:0}.c-nav--left,.c-nav--right{position:absolute;top:0;bottom:0}.c-nav--right{right:0;left:auto}.c-nav--fixed{position:fixed}.c-nav__item--active{background-color:#7b929e;color:#fff}.c-nav__item--brand:not(:disabled):hover{background-color:#2c3e50;color:#fff}.c-nav__item--brand:not(:disabled):focus{box-shadow:inset 0 0 0 2px #4dabf5}.c-nav__item--brand:not(:disabled):active{background-color:#1c2732;color:#fff}.c-nav__item--brand.c-nav__item--active{background-color:#2c3e50;color:#fff}.c-nav__item--info:not(:disabled):hover{background-color:#2196f3;color:#fff}.c-nav__item--info:not(:disabled):focus{box-shadow:inset 0 0 0 2px #4dabf5}.c-nav__item--info:not(:disabled):active{background-color:#0c7fda;color:#fff}.c-nav__item--info.c-nav__item--active{background-color:#2196f3;color:#fff}.c-nav__item--warning:not(:disabled):hover{background-color:#ff9800;color:#fff}.c-nav__item--warning:not(:disabled):focus{box-shadow:inset 0 0 0 2px #4dabf5}.c-nav__item--warning:not(:disabled):active{background-color:#d17d00;color:#fff}.c-nav__item--warning.c-nav__item--active{background-color:#ff9800;color:#fff}.c-nav__item--success:not(:disabled):hover{background-color:#4caf50;color:#fff}.c-nav__item--success:not(:disabled):focus{box-shadow:inset 0 0 0 2px #4dabf5}.c-nav__item--success:not(:disabled):active{background-color:#3e8f41;color:#fff}.c-nav__item--success.c-nav__item--active{background-color:#4caf50;color:#fff}.c-nav__item--error:not(:disabled):hover{background-color:#f44336;color:#fff}.c-nav__item--error:not(:disabled):foc
```

Comment:
let's include link in html: `<link rel="stylesheet" href="https://unpkg.com/blaze">`